src/promptflow/promptflow/_sdk/_telemetry/activity.py

```diff
@@ -34,6 +34,67 @@ class ActivityCompletionStatus(object):
 request_id_context = ContextVar("request_id_context", default=None)
 
 
+def log_activity_start(activity_info, logger) -> None:
```

Comment:
docstring on what's the type for activity_info?

Reply:
updated